src/components/Header/HeaderContent.tsx

```diff
@@ -52,10 +53,10 @@ const HeaderContent = ({ className }) => {
                             gap: "5px",
                         }}
                     />
-                </a>
+                </AletheiaButton>
             </Col>
             <Col lg={{ span: 16 }} md={{ span: 16 }} xs={{ span: 13 }}>
-                <a onClick={() => router.push("/")}>
+                <a href="/">
```

Comment:
Not a request for change: I wonder if we should use the nextjs `Link` for internal navigation.